views/layout.pug

```diff
@@ -0,0 +1,41 @@
+doctype html
+
+html
+	head
+		meta(charset="UTF-8")
+		meta(http-equiv="X-UA-Compatible" content="IE-edge, chrome=1")
+		meta(name="viewport" content="width=device-width, initial-scale=1")
+		title Styleguide Driven Development
+		link(rel="stylesheet", href="/css/core.0.12.0.css")
+		link(rel="stylesheet", href="/css/theme.css")
+	body
+		block head
+			.c-navbar
+				ul.c-navbar-menu
+					li.c-navbar-menu-item
+						a.c-navbar-menu-logo(href="/")
+							svg.c-navbar-menu-link-icon
```

Comment:
again i am going to refer you to https://codepen.io/hanakin/pen/dRoZzO as a reference for how this should function for icons